Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/BreakStrategyTests.cs

```diff
@@ -0,0 +1,167 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.PlatformConfiguration.AndroidSpecific;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[NUnit.Framework.Category(Core.UITests.UITestCategories.Label)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.None, 0, "[Enhancement] Label: Break Strategy on Android", PlatformAffected.Android)]
```

Comment:
What do you think about this change?
```suggestion
	[Issue(IssueTracker.Github, 10743, "[Enhancement] Label: Break Strategy on Android", PlatformAffected.Android)]
```
This way we can search this TestPage using the issue number.